Formula/iperf3.rb

```diff
@@ -5,6 +5,11 @@ class Iperf3 < Formula
   sha256 "bee427aeb13d6a2ee22073f23261f63712d82befaa83ac8cb4db5da4c2bdc865"
   license "BSD-3-Clause"
 
+  livecheck do
+    url :homepage
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    url "https://downloads.es.net/pub/iperf/"
    regex(/href=.*?iperf[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Unless there's a specific reason to check the GitHub repository tags (other than it simply being the default check here), it would arguably be more appropriate to check the same source as the `stable` URL.